src/Api/Client.php

```diff
@@ -127,10 +124,43 @@ public function call($endPoint, $httpRequestMethod, $content = [])
 		return $response;
 	}
 
+    /**
+     * @param string
+     * @return resource|false
+     */
+    private function getCurlHandle($endPoint)
+    {
+        $url = self::URL.$endPoint;
+        if (array_key_exists($endPoint, $this->curlHandles)) {
+            if (!isset($this->curlHandles[$endPoint])) {
+                $this->curlHandles[$endPoint] = curl_init($url);
+            }
+            
+            return $this->curlHandles[$endPoint];
+        }
+        
+        return false;
+    }
+	
+	public function closeAll() {
```

Comment:
A better naming would be `closeConnections()`

Reply:
done